### PR TITLE
Frontend volunteer profile

### PR DESCRIPTION
**Pull request by jansm04**
Created the volunteer profile page.
- added `p_name`, `pronouns`, `phone_number`, `city` and `province` to the `volunteers` table.
- created a `volunteer_profile_pics` table to store profile pictures as LONGBLOBs.
- added POST, GET, PUT and DELETE endpoints for profile pictures, using `multer` middleware to easily manage file uploads.
- profile pictures are stored using their binary representations.
- implemented volunteer profile card / form component (updatable fields: preferred name, pronouns, phone number and profile picture).
- implemented availability grid (days are stored as integers, with 0 representing Monday, up to 6 representing Sunday).

### CODE REVIEW

**Comment by Mageacallll** [comment]
Reminder in case, this PR follows the latest database schema of which the current AWS database has not implemented yet (pronouns, phone, location and date joined).

File: frontend/src/pages/volunteerProfile/index.js [hunk 1]
```diff
@@ -1,21 +1,67 @@
...
+                                <VolunteerDetailsCard volunteer={volunteer} />
+                            </div>    
+                            <div className="availability-card">
+                                <AvailabilityGrid volunteerId={volunteer.volunteer_id} availability={availability} setAvailability={setAvailability} />
+                            </div>
+                        </div>
+                        <div className="column-2">
+                            <div className="password-card">
+                                <ChangePasswordCard volunteer={volunteer} />
+                            </div>
```
Comment (jjessieshang) [inline frontend/src/pages/volunteerProfile/index.js:54]: Can remove the commented out code in this file

File: frontend/src/pages/volunteerProfile/index.js [hunk 2]
```diff
@@ -1,21 +1,67 @@
 // volunteer profile page
 import "./index.css";
-import React from 'react'
+import React, { useState } from 'react';
 import VolunteerLayout from "../../components/volunteerLayout";
-import UserProfileForm from "../../components/userProfileForm/UserProfileForm";
+import { fetchVolunteerData, getProfilePicture } from "../../api/volunteerService";
+import VolunteerDetailsCard from "../../components/volunteerProfile/volunteerDetailsCard";
+import ChangePasswordCard from "../../components/volunteerProfile/changePasswordCard";
+// import ClassPreferencesCard from "../../components/volunteerProfile/classPreferencesCard";
```
Comment (jjessieshang) [inline frontend/src/pages/volunteerProfile/index.js:8]: remove comments

**Review by jjessieshang — COMMENTED** [review]
Left some notes!

**Comment by jjessieshang** [comment]
Also, for consistency with the backend: let's match the days numbering defined here: https://github.com/ubclaunchpad/neuron/blob/051066f1e2251be7a702c9b30864c62a076472ca/backend/src/common/interfaces.ts#L35 which has monday as 1 and sunday as 6!

**Review by jjessieshang — APPROVED** [review]
Looks great!! PLEASE use the squash and merge feature when you merge :)

**Pull request merged**